deploy/kubernetes/resources/chirper/activity-stream-impl-service-akka-remoting.json

```diff
@@ -0,0 +1,22 @@
+{
+  "apiVersion": "v1",
+  "kind": "Service",
+  "metadata": {
+    "labels": {
+      "app": "activityservice"
+    },
+    "name": "activityservice-akka-remoting"
+  },
+  "spec": {
+    "ports": [
+      {
+        "port": 2551,
+        "protocol": "TCP",
+        "targetPort": 2551
+      }
+    ],
+    "selector": {
+      "app": "activityservice"
+    }
+  }
+}
```

Comment:
Is there any plan to generate all this configuration automatically similar to what is a achiever with a ConductR bundle?